DEATHS_extract_processing.R

```diff
@@ -0,0 +1,63 @@
+
+#####################################################
+# Draft DEATHS Extract Processing Code
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Takes IT extract of Deaths data.
+#               Creates new death date variable,
+#               uses NRS death date as death date if avaliable.
+#               Saves outputted flagged dataset.
+#####################################################
+
+
+## packages ##
+
+library(dplyr)
+library(stringr)
+library(readr)
+
+
+
+## Read data in ##
+data <- readr::read_csv(get_it_deaths_path())
```

Comment:
I've removed this line as you have it underneath and also just removed spaces within the script. But looks good to me :)